engine/src/ibmp.cpp

```diff
@@ -2902,6 +2908,8 @@ bool MCImageEncodeRawTrueColor(MCImageBitmap *p_bitmap, IO_handle p_stream, Expo
 		case EX_RAW_ABGR:
 			MCBitmapConvertRow<EX_RAW_ABGR>(t_row_buffer, (uint32_t*)t_src_ptr, p_bitmap->width);
 			break;
+        default:
+            break;
```

Comment:
This should be `MCUnreachableReturn(false)`.  If you just break here then the following `IO_write()` will output an uninitialised buffer.